samples/test-samples.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script requires the variables SOURCE_DIR, BUILD_DIR, and
+# PREFIX_DIR to be defined:
+#
+#     cmake -DSOURCE_DIR=~/openenclave -DBUILD_DIR=~/openenclave/build -DPREFIX_DIR=/opt/openenclave -P ~/openenclave/samples/test-samples.cmake
+
+# The samples cannot run in simulation mode.
```

Comment:
FWIW: Some samples can be ran in simulation mode (i.e. https://github.com/Microsoft/openenclave/blob/master/samples/helloworld/Makefile#L16)

Reply:
Ah, interesting. Thank you; I will let you separate them out and get what does work in simulation working 😄